packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -5100,10 +5256,17 @@ The IdP Authorization Server (AS) endpoints are currently defined as part of the
 
```

Comment:
nit on the previous line: shouldn't it now read `##### OAuth 2.0 and OpenID Connect Credentials object` (removed `Client`)?